### DISCUSSION TITLE
Can Liquorix be made operational on unsupported Linux distributions?

### DISCUSSION THREAD

**Discussion by Ricky-Tigg** [damentz/liquorix-package#154] [Q&A] [answered]
Hello. As this project leads to the sole creation of a **binary file** in **DEB format**, the temptation for a user to experiment with the aim of getting this Liquorix kernel operational in a Linux distribution whose packaging format is not supported by this project, is pertinent. Experiment with [Alien](https://wiki.debian.org/Alien), a tool which operates conversions of an installer as binary file of a Linux distribution packaging system format into an other one of a Linux distribution, reveals under conditions relevant while no other viable method – **build from source**, **universal software package open source format**, abusively presented as being universal, since as practices demonstrate it, neither AppImage, nor Snap and Flatpak reveal so to be. – is at user's disposal to achieve this goal. Yet Alien, despite being an highly valuable project, must have limitations that makes it experimental and issues. Needless to say that experimenting on a virtual machine is a relevant precaution – Qemu-KVM would do.

It can reasonably be assumed that
- Liquorix must be composed of important system packages that are essential for the functioning of the system.
- Packages might be set up differently by Debian and a non-DEB Linux distribution, and it is known that packages from different Linux distributions cannot be used interchangeably.

Supposing that such a conversion is not only successful but also revealing itself operational, the resulting **binary file** would not for reason be eligible for support from this project. Then this attempt would not be worth the effort. Despite is seems desperate, I have reason to believe that there might be a developer of this project, able to bring instruction to achieve this goal.

PS: I am myself willing to get Liquorix operational on Linux distributions whose packaging format is RPM, among others.

**Comment by damentz** [comment, marked as answer]
I think the right way to go is to make a package in Fedora COPR: https://copr.fedorainfracloud.org/coprs/.  If you search for "lqx", there's a few packages and forks out there, but no one appears to be seriously maintaining Liquorix there anymore.
